autobot/telegram/parser.py

```diff
@@ -238,16 +238,139 @@ def _entity_parser (entity:dict):
         return msg_obj
 
     # methods to parse games objects
+    def _parse_game (self,key:str,val:dict) -> Game|GameHighScore:
+        game_obj = None
+
+        if (key == "game"):
+            game_obj = Game()
+            for k,v in val.items():
+                match k:
+                    case "photo":
+                        pht = self._parse_photosize(k,v)
+                        setattr(game_obj,k,pht)
+                    case "animation":
+                        anim = self._parse_animation(k,v)
+                        setattr(game_obj,k,anim)
+                    case _:
+                        setattr(game_obj,k,v)
+
+        elif (key == "game_high_score"):
+            game_obj = GameHighScore()
+            for k,v in val.items():
+                setattr(game_obj,k,v)
+
+        return game_obj
     
     
     # methods to parse inline objects
+    def _parse_inline (self,key:str,val:dict) -> InlineKeyboardMarkup|InlineQueryResult|InputMessageContent:
```

Comment:
So I'm thinking you're trying to break down the parsing of objects in inline to small related objects. This function name according to the current semantics of the codebase would mean that this function is responsible for parsing every object in the inline module, which I don't think it's what it's doing according to the function body.

So I'd suggest you write a function such as `_parse_inlinequeryresults` which parses all objects related to InlineQueryResult. Then write another function such as `_parse_inputemessagecontents` to parse those objects and then also `_parse_inlinekeyboardobjects` for objects related to that.
Does this approach make sense?